### PR TITLE
Added sentry to docs

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

File: docs/errorhandling.rst [hunk 1]
```diff
@@ -28,6 +28,46 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+Error Logging Tools
+-------------------
+
+Sending error mails, even if just for critical ones, can become
+overwhelming if enough users are hitting the error and log files are
+typically never looked at. This is why we're recommending using
```
Comment (ThiefMaster) [inline docs/errorhandling.rst]: *we recommend*

File: docs/errorhandling.rst [hunk 2]
```diff
@@ -28,6 +28,46 @@ exception to the :attr:`~flask.Flask.logger`.
...
+`Sentry <http://www.getsentry.com/>`_ for dealing with application errors.
+It's available as an Open Source project `on GitHub
+<github.com/getsentry/sentry>`__ and is also available as `Hosted Version
```
Comment (ThiefMaster) [inline docs/errorhandling.rst]: *as a hosted version*

File: docs/errorhandling.rst [hunk 3]
```diff
@@ -28,6 +28,46 @@ exception to the :attr:`~flask.Flask.logger`.
...
+<https://getsentry.com/signup/>`_ which you can try for free. Sentry
+aggregates duplicate erorrs, captures the full stack trace and local
```
Comment (ThiefMaster) [inline docs/errorhandling.rst]: *errors*

File: docs/errorhandling.rst [hunk 4]
```diff
@@ -28,6 +28,46 @@ exception to the :attr:`~flask.Flask.logger`.
...
+variables for debugging, and send you mails based on new errors or
```
Comment (ThiefMaster) [inline docs/errorhandling.rst]: *sends*

**Comment by RonnyPfannschmidt** [comment]
i do wonder if the affiliation of @mitsuhiko with sentry should be mentioned as part of that section

**Comment by ThiefMaster** [comment]
Considering it's open source and not just a commercial service I don't see a big need for it - especially if other contributors :+1: this PR here anyone who complains later about subtle advertising can be pointed here seeing that others agree with the idea of mentioning it.

IMHO this kind of disclosure would be merely noise to the typical user reading the Flask docs - it's not really important that the person who wrote the paragraph is involved with sentry

**Comment by mitsuhiko** [comment]
JFTR: the suggestion to have sentry in the docs is quite old and predates me working with the Sentry team. 0.11 pushes error logging by default and as a side effect of this I figured it might make sense to send people on the path of proper error monitoring.

**Pull request merged**